doc/man7/OSSL_PROVIDER-FIPS.pod

```diff
@@ -29,14 +29,16 @@ L<EVP_MD_fetch(3)> or L<EVP_CIPHER_fetch(3)>, as well as with other
 functions that take a property query string, such as
 L<EVP_PKEY_CTX_new_from_name(3)>.
 
-It isn't mandatory to query for any of these properties, except to
-make sure to get implementations of this provider and none other.
-
-The C<fips=yes> property can be use to make sure only FIPS approved
-implementations are used for crypto operations.  This may also include
-other non-crypto support operations that are not in the FIPS provider,
-such as asymmetric key encoders,
-see L<OSSL_PROVIDER-default(7)/Asymmetric Key Management>.
+To be FIPS compliant, it is mandatory to include C<fips=yes> as
+part of all property queries.  This ensures that only FIPS approved
+implementations are used for cryptographic operations.  The C<fips=yes>
+query may also include other non-crypto support operations that
+are not in the FIPS provider, such as asymmetric key encoders, see
+L<OSSL_PROVIDER-default(7)/Asymmetric Key Management>.
+
+It is not mandatory to include C<provider=fips> as part of your property
+query.  Doing so guarantees that the OpenSSL FIPS provider is used for
```

Comment:
Maybe state it again in the second sentence here again.. Maybe a bit confusing otherwise after 'it is not' from the first sentence...

Reply:
done